### ISSUE TITLE
Actual: Non-admins are able to make their apps ‘public’ after creating apps using upload .zip file option. Expected: Only admins should be able to make the apps public.

### ISSUE THREAD

**Issue by jasneetkaurdeloitte** [SEMOSS/Semoss#341]
Actual: Non-admins are able to make their apps ‘public’ after creating apps using upload .zip file option.
Expected: Only admins should be able to make the apps public.

**Comment by themaherkhalil** [comment]
The api will now throw an error when attempting to do this. But we should also hide the ability all together on the UI if the user is not an admin

**Comment by jasneetkaurdeloitte** [comment]
@themaherkhalil @neelneelneel  Non-admins are still able to make their apps public after creating apps using upload .zip file option in the global environment.

**Comment by themaherkhalil** [comment]
It is fixed - it has not been pushed to that environment.  Once the ticket is resolved the logic in GitHub will auto close the ticket. Release to a specific environment is not something that can be tracked from the board
